src/components/TopPicks/TopPicks.tsx

```diff
@@ -120,3 +162,34 @@ export const TopPick = ({ comment }: { comment: CommentType }) => (
     </div>
   </div>
 );
+
+export const TopPicks = ({ comments }: { comments: Array<CommentType> }) => {
+  const leftColComments: Array<CommentType> = [];
+  const rightColComments: Array<CommentType> = [];
+  comments.forEach((comment, index) =>
+    index % 2 === 0
+      ? leftColComments.push(comment)
+      : rightColComments.push(comment)
+  );
+  return (
+    <div className={picksWrapper}>
+      <div className={twoColComments}>
+        <div className={cx(columWrapperStyles, colLeft)}>
+          {leftColComments.map(comment => (
+            <TopPick comment={comment} />
+          ))}
+        </div>
+        <div className={cx(columWrapperStyles, colRight)}>
+          {rightColComments.map(comment => (
+            <TopPick comment={comment} />
+          ))}
+        </div>
+      </div>
+      <div className={oneColComments}>
+        {comments.map(comment => (
+          <TopPick comment={comment} />
+        ))}
+      </div>
+    </div>
```

Comment:
This is great. It's really easy to see what's happening and whilst there's duplication of the picks on the dom, the win in terms of simplicity is nice